README.md

```diff
@@ -67,18 +67,17 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 
 * Code in a functional way. Avoid mutation (side effects) when you can.
 
-* Do not program defensively (see
-  [http://www.erlang.se/doc/programming_rules.shtml#HDR11](http://www.erlang.se/doc/programming_rules.shtml#HDR11)).
+* [Avoid defensive programming](http://www.erlang.se/doc/programming_rules.shtml#HDR11)
 
-* Do not mutate arguments unless that is the purpose of the method.
+* Keep method arguments unchanged.
```

Comment:
I think this could be phrased using `avoid`, as you've done in other places
```suggestion
* Avoid mutating arguments, unless that is the purpose of the method.
```

Reply:
Applied! Except the "unless that is the purpose of the method" part since we shouldn't be appending "except when it makes sense to break the rule" because that's implicit.